comments.php

```diff
@@ -58,8 +67,23 @@
 		</ol><!-- .comment-list -->
 
 		<?php
+		if ( _s_using_amp_live_list_comments() ) {
+			add_filter( 'navigation_markup_template', '_s_add_amp_live_list_pagination_attribute' );
```

Comment:
This is a little awkward. It would be preferable if AMP allowed for a `nav` element (or `role="navigation"`) instead of requiring an element with a custom `pagination` attribute.